Import error generating new app as API only

I'm seeing issues similar to #492 using v0.9.4. 
`actions/render.go:5:2: imported and not used: "github.com/gobuffalo/packr"`

It appears to be related to newapp/templates/actions/render.go.tmpl importing packr in API mode, where the packr use is left out when in API mode.

I've got a fix I'll put a PR in for in just a minute to get this sorted out. Hope that works!

--- a/generators/newapp/templates/actions/render.go.tmpl
+++ b/generators/newapp/templates/actions/render.go.tmpl
@@ -1,7 +1,9 @@
 package actions
 
 import (
+  {{ if .asWeb -}}
   "github.com/gobuffalo/packr"
+  {{ end -}}
   "github.com/gobuffalo/buffalo/render"
 )
 

